src/inputcard.ts

```diff
@@ -238,6 +237,7 @@ document.addEventListener('DOMContentLoaded', () => {
               myRes => (myRes.status === 200 ? JSON.stringify(myRes.value.data) : 'fakeWallet'),
             );
             sessionStorage.setItem('wallet', walletResp);
+            sessionStorage.setItem('securityCode', (creditcardformSecurecode as HTMLInputElement).value);
```

Comment:
it can be done?

Reply:
@pasqualespica We have to assess it during the sprint, but it's needed since the cvv is a required input to `pay`